src/Main.java

```diff
@@ -0,0 +1,28 @@
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
This could be a class itself to be called in Main (still too long for a class).

Reply:
our mentor suggested that have whole of this part of code in main method